### PR TITLE
Add missing tokens and update sardine logo

### PR DESCRIPTION
**Pull request by muliswilliam**
Resolves https://github.com/brave/brave-browser/issues/25029, https://github.com/brave/brave-browser/issues/24899

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [x] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [x] New files have MPL-2.0 license header
- [x] Adequate test coverage exists to prevent regressions
- [x] Major classes, functions and non-trivial code blocks are well-commented
- [x] Changes in component dependencies are properly reflected in `gn`
- [x] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [x] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### Light Mode
Screenshot 2022-08-30 at 09 14 07

### Dark Mode
Screenshot 2022-08-30 at 09 14 42

https://user-images.githubusercontent.com/48117473/187372214-0d518f05-d409-455c-9331-28ef56f77e62.mov

### CODE REVIEW

File: components/brave_wallet/browser/brave_wallet_constants.cc [hunk 1]
```diff
@@ -163,78 +163,89 @@ const std::vector<mojom::BlockchainToken>& GetRampBuyTokens() {
...
+        mojom::CoinType::ETH},
+       {"0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48", "USD Coin", "usdc.png",
+        true, false, "USDC", 6, true, "", "", mojom::kMainnetChainId,
+        mojom::CoinType::ETH},
+       {"0xe41d2489571d322189246dafa5ebde1f4699f498", "Ox", "zrx.png", true,
+        false, "ZRX", 6, true, "", "", mojom::kMainnetChainId,
+        mojom::CoinType::ETH},
+       {"0xbb0e17ef65f82ab018d8edd776e8dd940327b28b", "Axie Infinity",
+        "axs.png", true, false, "AXS", 6, true, "", "", mojom::kMainnetChainId,
+        mojom::CoinType::ETH}});
```
Comment (onyb) [inline components/brave_wallet/browser/brave_wallet_constants.cc:248]: Please review all the decimals correctly. ZRX and AXS should have `6`.

**Review by onyb — CHANGES_REQUESTED** [review]

File: components/brave_wallet/browser/brave_wallet_constants.cc (hunk 1 above)
Comment (muliswilliam) [reply components/brave_wallet/browser/brave_wallet_constants.cc]: @onyb  Fixed here: 4027fca7790bb2df255964f4fd6f23c98632a743

**Review by onyb — APPROVED** [review]
++

**Comment by srirambv** [comment]
Verification passed on 

Brave | 1.45.25 Chromium: 105.0.5195.102 (Official Build) nightly (64-bit) 
-- | --
Revision | `4c16f5ffcc2da70ee2600d5db77bed423ac03a5a-refs/branch-heads/5195_55@{#4}`
OS | Windows 11 Version 21H2 (Build 22000.856)

- Verified steps from PR
- Verified missing tokens are now listed
- Verified Buy via Sardine option is shown for supported tokens
- Verified on both light and dark theme
- Verified Sardine is only shown for US locale
- Encountered https://github.com/brave/brave-browser/issues/25158

https://user-images.githubusercontent.com/17010094/188398243-7b4d54b1-bc9a-45c6-9850-5c022d77b1f8.mp4

**Pull request merged**